provider/docker/docker.go

```diff
@@ -711,6 +713,31 @@ func (p *Provider) getBasicAuth(container dockerData) []string {
 	return []string{}
 }
 
+func (p *Provider) getRequestHeaders(container dockerData) map[string]string {
+	h := make(map[string]string)
+	if requestHeaders, err := getLabel(container, types.LabelFrontendRequestHeader); err == nil {
+		for _, v := range strings.Split(requestHeaders, ",") {
+			k := strings.Split(v, ":")
+			h[k[0]] = k[1]
```

Comment:
We will crash if the split in the previous line yields a single line only. Should we have some extra sanity check?

Reply:
Will do!